_data/simple-icons.json

```
@@ -487,6 +487,11 @@
             "hex": "A5915F",
             "source": "https://www.altium.com/altium-designer/"
         },
+        {
+            "title": "Alwaysdata",
+            "hex": "E9568E",
+            "source": "https://www.alwaysdata.com/en/references/"
```

```suggestion
            "source": "https://www.alwaysdata.com"
```

Seems this URL does not contain any logo resource. Let's use the homepage instead.